src/util/notifications_util.js

```diff
@@ -145,6 +147,9 @@ class Notificator {
                 }
             } catch (err) {
                 dbg.error("Failed to notify. err = ", err, ", str =", str);
+                if (this.nc_config_fs) {
+                    new NoobaaEvent(NoobaaEvent.NOTIFICATION_FAILED).create_event(null, {name: notif?.meta?.name}, err);
```

Comment:
I saw in other implementations that they used `undefined` instead of `null`.
```suggestion
                    new NoobaaEvent(NoobaaEvent.NOTIFICATION_FAILED).create_event(undefined, {name: notif?.meta?.name}, err);
```

In case the name is `undefined` (`notif?.meta?.name`) would you want to pass additional arguments?

Reply:
I've switched to `underfined`.

I don't think more details are needed. `name` is not available in an unlikely case.